### ISSUE TITLE
Add into_model() for Insert/Update/Delete

### ISSUE THREAD

**Issue by nicoulaj** [SeaQL/sea-orm#148]
It is a common need to insert/update/delete and select some return columns as a single query. Currently sea-orm does not make this easy, so I think having ideally `into_model()` for Insert/Update/Delete would be very convenient. What do you think ?

**Comment by nicoulaj** [comment]
related: SeaQL/sea-query#120

**Comment by billy1624** [comment]
1. I'm thinking how we handle databases that don't support `returning`? Should we just return the original `ActiveModel` passed into insert / update / delete?

2. Also, what will be `returned` if we insert / update / delete multiple records? Or this `into_model()` method is not supported for insert / update / delete multiple records.

**Comment by nicoulaj** [comment]
> I'm thinking how we handle databases that don't support `returning`? Should we just return the original `ActiveModel` passed into insert / update / delete?

Indeed, it looks like this is supported by postgres, sqlite and mariadb but not mysql... Maybe use a feature flag ?

> Also, what will be `returned` if we insert / update / delete multiple records? Or this `into_model()` method is not supported for insert / update / delete multiple records.

I don't see how this is different from normal select ? `into_model()` returns a `Selector`, `Selector::all()` returns a `Vec`. But maybe I am missing something.

**Comment by nicoulaj** [comment]
I think this issue can be closed since #183 handled it ?
